diff --git a/models/bbu/hokuyo_noisy/model.sdf b/models/bbu/hokuyo_noisy/model.sdf
index f0bbde7..f637a60 100644
--- a/models/bbu/hokuyo_noisy/model.sdf
+++ b/models/bbu/hokuyo_noisy/model.sdf
@@ -9,7 +9,7 @@
       <visual name="visual">
         <geometry>
           <mesh>
-            <uri>model://hokuyo/meshes/hokuyo.dae</uri>
+            <uri>model://hokuyo_noisy/meshes/hokuyo.dae</uri>
           </mesh>
         </geometry>
       </visual>
diff --git a/models/bbu/robotino_bbu/model.sdf b/models/bbu/robotino_bbu/model.sdf
index 581648a..a81f42b 100644
--- a/models/bbu/robotino_bbu/model.sdf
+++ b/models/bbu/robotino_bbu/model.sdf
@@ -60,12 +60,16 @@
       </axis>
     </joint>
 
-    <plugin name="Odometry" filename="libodometry.so"/>
 
-
-	<!--plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
-		<robotNamespace>/robotino_bbu1</robotNamespace>
-	</plugin-->
+                <!-- 差動駆動システムのプラグイン -->
+            <plugin filename="gz-sim-diff-drive-system" name="gz::sim::systems::DiffDrive">
+                <topic>cmd_vel</topic>    <!-- 速度指令のトピック名 -->
+                <left_joint>left_wheel_joint</left_joint>    <!-- 左車輪のジョイント名 -->
+                <right_joint>right_wheel_joint</right_joint>    <!-- 右車輪のジョイント名 -->
+                <wheel_separation>0.35</wheel_separation>    <!-- 車輪間距離 -->
+                <wheel_radius>0.1</wheel_radius>    <!-- 車輪半径 -->
+                <odom_publish_frequency>50</odom_publish_frequency>    <!-- Odometry公開の頻度 -->
+	</plugin>
 
   </model>
 </sdf>
diff --git a/models/machine/model.sdf b/models/machine/model.sdf
index 6e60fdd..95b9624 100644
--- a/models/machine/model.sdf
+++ b/models/machine/model.sdf
@@ -23,11 +23,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-	    <uri>file://media/materials/scripts/gazebo.material</uri>
-	    <name>Gazebo/Grey</name>
-	  </script>
-	</material>
+	  <ambient>0.5 0.5 0.5 1</ambient>
+	  <diffuse>0.7 0.7 0.7 1</diffuse>
+	  <specular>0.1 0.1 0.1 1</specular>
+	  <emissive>0 0 0 1</emissive>
+        </material>
       </visual>
       <collision name="low_plate_collision">
         <pose>0 0 0.03 0 0 0 </pose>
@@ -46,11 +46,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-	    <uri>file://media/materials/scripts/gazebo.material</uri>
-	    <name>Gazebo/Grey</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
       <collision name="middle_plate_collision">
         <pose>0.02 0 0.06 0 0 0 </pose>
@@ -69,11 +69,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-	    <uri>file://media/materials/scripts/gazebo.material</uri>
-	    <name>Gazebo/Grey</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
       <collision name="uppest_plate_collision">
         <pose>0.02 0 0.13 0 0 0 </pose>
@@ -92,11 +92,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-	    <uri>file://media/materials/scripts/gazebo.material</uri>
-	    <name>Gazebo/Grey</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
     <!-- only two sticks, I don't have triangles -->
       <collision name="upper_plate_left_collision">
@@ -116,11 +116,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-	    <uri>file://media/materials/scripts/gazebo.material</uri>
-	    <name>Gazebo/Grey</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
       <collision name="upper_plate_right_collision">
         <pose>0.04 0.05 0.095 -0.86 0 0 </pose>
@@ -139,11 +139,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-	    <uri>file://media/materials/scripts/gazebo.material</uri>
-	    <name>Gazebo/Grey</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
     <!-- no cable -->
       <collision name="rfid_collision">
@@ -163,11 +163,11 @@
           </box>
         </geometry>
 	<material>
-	  <script>
-            <uri>file://media/materials/scripts/gazebo.material</uri>
-            <name>Gazebo/Black</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
       <collision name="trafficlight_collision">
         <pose>0.02 0 0.205 0 0 0 </pose>
@@ -188,11 +188,11 @@
           </cylinder>
         </geometry>
 	<material>
-	  <script>
-            <uri>file://media/materials/scripts/gazebo.material</uri>
-            <name>Gazebo/Black</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
       <visual name="red">
         <pose>0.02 0 0.253 0 0 0 </pose>
@@ -282,11 +282,11 @@
           </cylinder>
         </geometry>
 	<material>
-	  <script>
-            <uri>file://media/materials/scripts/gazebo.material</uri>
-            <name>Gazebo/Black</name>
-	  </script>
-	</material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
       </visual>
     </link>
   </model>
diff --git a/models/mps_base/model.sdf b/models/mps_base/model.sdf
index c3cf5f5..dd705c4 100644
--- a/models/mps_base/model.sdf
+++ b/models/mps_base/model.sdf
@@ -26,12 +26,11 @@
           </mesh>
         </geometry>
 	<material>
-          <script>
-            <uri>model://mps_base/materials/scripts</uri>
-            <uri>model://mps_base/materials/textures</uri>
-            <name>mps_base</name>
-          </script>
-        </material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
         <cast_shadows>false</cast_shadows>
       </visual>
       <!-- Collision Elements: -->
diff --git a/models/mps_cap/model.sdf b/models/mps_cap/model.sdf
index b823f7a..b07b177 100644
--- a/models/mps_cap/model.sdf
+++ b/models/mps_cap/model.sdf
@@ -26,12 +26,11 @@
           </mesh>
         </geometry>
 	<material>
-          <script>
-            <uri>model://mps_cap/materials/scripts</uri>
-            <uri>model://mps_cap/materials/textures</uri>
-            <name>mps_cap</name>
-          </script>
-        </material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
         <cast_shadows>false</cast_shadows>
       </visual>      <!-- Shelf: -->
       <collision name="collision_shelf">
diff --git a/models/mps_delivery/model.sdf b/models/mps_delivery/model.sdf
index 273367d..2b1f213 100644
--- a/models/mps_delivery/model.sdf
+++ b/models/mps_delivery/model.sdf
@@ -26,12 +26,11 @@
           </mesh>
         </geometry>
 	<material>
-          <script>
-            <uri>model://mps_delivery/materials/scripts</uri>
-            <uri>model://mps_delivery/materials/textures</uri>
-            <name>mps_delivery</name>
-          </script>
-        </material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
         <cast_shadows>false</cast_shadows>
       </visual>
 
diff --git a/models/mps_ring/model.sdf b/models/mps_ring/model.sdf
index 54784eb..528be76 100644
--- a/models/mps_ring/model.sdf
+++ b/models/mps_ring/model.sdf
@@ -26,12 +26,11 @@
           </mesh>
         </geometry>
 	<material>
-          <script>
-            <uri>model://mps_ring/materials/scripts</uri>
-            <uri>model://mps_ring/materials/textures</uri>
-            <name>mps_ring</name>
-          </script>
-        </material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
         <cast_shadows>false</cast_shadows>
       </visual>
       
diff --git a/models/mps_storage/model.sdf b/models/mps_storage/model.sdf
index c8967ad..9a2d060 100644
--- a/models/mps_storage/model.sdf
+++ b/models/mps_storage/model.sdf
@@ -26,12 +26,11 @@
           </mesh>
         </geometry>
 	<material>
-          <script>
-            <uri>model://mps_storage/materials/scripts</uri>
-            <uri>model://mps_storage/materials/textures</uri>
-            <name>mps_storage</name>
-          </script>
-        </material>
+  <ambient>0.5 0.5 0.5 1</ambient>
+  <diffuse>0.7 0.7 0.7 1</diffuse>
+  <specular>0.1 0.1 0.1 1</specular>
+  <emissive>0 0 0 1</emissive>
+</material>
         <cast_shadows>false</cast_shadows>
       </visual>
       <!-- Collision Elements: -->
diff --git a/models/robotino_gripper/model.sdf b/models/robotino_gripper/model.sdf
index ff51d91..1b4f435 100644
--- a/models/robotino_gripper/model.sdf
+++ b/models/robotino_gripper/model.sdf
@@ -24,7 +24,7 @@
         </geometry>
         <cast_shadows>false</cast_shadows>
       </visual>
-      <collision name="collision">
+      <collision name="collisionA">
         
         <geometry>
 		    <box>
@@ -57,7 +57,7 @@
         <cast_shadows>false</cast_shadows>
       </visual>
 
-        <collision name="collision">
+        <collision name="collisionB">
         	<pose>-0.014 0.006 0 0 0 0</pose>
             <geometry>
                 <box>
@@ -65,7 +65,7 @@
                 </box>
             </geometry>
         </collision>
-        <collision name="collision">
+        <collision name="collisionC">
         	<pose>-0.023 0.021 0 0 0 0</pose>
             <geometry>
                 <box>
@@ -99,7 +99,7 @@
         <cast_shadows>false</cast_shadows>
       </visual>
 
-        <collision name="collision">
+        <collision name="collisionD">
         	<pose>-0.014 0.006 0 0 0 0</pose>
             <geometry>
                 <box>
@@ -107,7 +107,7 @@
                 </box>
             </geometry>
         </collision>
-        <collision name="collision">
+        <collision name="collisionE">
         	<pose>-0.023 0.021 0 0 0 0</pose>
             <geometry>
                 <box>
diff --git a/worlds/bbu_2015.world b/worlds/bbu_2015.world
index ba61311..d58893f 100644
--- a/worlds/bbu_2015.world
+++ b/worlds/bbu_2015.world
@@ -14,7 +14,7 @@
 		</include>
 
 		<include>
-		  <uri>model://logistics_league_field</uri>
+		  <uri>model://logistics_league_field_2017</uri>
       	  <pose>0 2.5 0 0 0 0</pose>
 		</include>
 
@@ -35,6 +35,34 @@
       	  <pose>-5.5 -0.5 0 0 0 3.1415</pose>
 		</include>-->
 
+		<!-- Robotinos  -->
+		<!-- Team Cyan -->
+		<include>
+		  <uri>model://robotino-nr-1</uri>
+		  <pose>4.5 0.5 0 0 0 1.57</pose>
+	  	</include>
+		<include>
+		  <uri>model://robotino-nr-2</uri>
+		  <pose>5.5 0.5 0 0 0 1.57</pose>
+	  	</include>
+		<include>
+		  <uri>model://robotino-nr-3</uri>
+		  <pose>6.5 0.5 0 0 0 1.57</pose>
+	  	</include>
+		<!-- Team Magenta -->
+		<include>
+		  <uri>model://robotino-nr-4</uri>
+		  <pose>-4.5 0.5 0 0 0 1.57</pose>
+	  	</include>
+		<include>
+		  <uri>model://robotino-nr-5</uri>
+		  <pose>-5.5 0.5 0 0 0 1.57</pose>
+	  	</include>
+		<include>
+		  <uri>model://robotino-nr-6</uri>
+		  <pose>-6.5 0.5 0 0 0 1.57</pose>
+	  	</include>
+
 		<include>
 		  <name>C-BS</name>
 		  <uri>model://mps_base</uri>
